python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -435,6 +439,8 @@ cdef class ParquetWriter:
         self.comp_type = _get_comp_type(compression)
         self.index = index
         self.initialized = False
+        self.row_group_size_bytes = row_group_size_bytes or 0
+        self.row_group_size_rows = row_group_size_rows or 0
```

Comment:
I have found that sometimes `val or default` constructions get awkward. There are a lot of values besides `None` that would evaluate as falsy, like `[]` or `""` or `False` which would all be semantically different than the default of `None` but would result in a 0 value here (which is later reinterpreted as "falsy" via the `if self.row_group_size_bytes`...). I wonder if there's a better choice here that is more consistent with the behavior below.

Reply:
Can we not just type `row_group_size_*` arguments as `int`  in the signature and default to  `0`?

Reply:
I'm happy with the present state of the design. I agree that if we document the default value at the Python layer, we should enforce it at the Python layer. Overriding C++ may be the only self-explanatory option.